kafka/0.5/Dockerfile

```diff
@@ -35,11 +43,9 @@ RUN echo "$MD5HASH  /tmp/kafka.tgz" | md5sum -c - &&\
 COPY ./log4j.properties $KAFKA_HOME/config/log4j.properties
 
 #
-# Change ownership and switch user
+# Allow random UID to use Kafka
 #
-RUN chown -R kafka $KAFKA_HOME && \
-    chgrp -R kafka $KAFKA_HOME
-USER kafka
+RUN chmod -R g+w,o+w $KAFKA_HOME
```

Comment:
So it's [known behavior](https://github.com/moby/moby/issues/6119) of Docker: the COPY above won't run in the context of the previously set user, causing `log4j.properties` to be owned by root, naturally causing `chmod` to fail here. So the user switch to kafka must happen later. Meh.